index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
```

Comment:
**Kindly do these changes:**
- Form uses the action and method attribute to integrate with formspree service.

Reply:
Integrated and was submitted successfully.